src/Archive/Parsers/DefaultParserFactory.ts

```diff
@@ -40,7 +40,7 @@ export class DefaultParserFactory implements ParserFactory {
       case SpecType.POSTMAN:
         return new BaseParser(this.postmanValidator);
       default:
-        return null;
+        throw new Error('Incorrect a specification type');
```

Comment:
please keep this fix "as is" but further move all unrelated fixes to separate PRs to have clean narrow PR scope, minimize possible interference with another PRs, minimize review time, simplify QA, etc.

Reply:
@mvhysko said keep it

Reply:
i wrote the same "keep" above, but it's an exception only for the first PR